packages/runtime/src/account.ts

```diff
@@ -39,8 +39,12 @@ export class AccountStore implements AccountStoreI {
   createdApps: Map<number, SSCAttributesM>;
   createdAssets: Map<number, AssetDef>;
 
-  constructor (balance: number | bigint, account?: Account) {
-    if (account) {
+  constructor (balance: number | bigint, account?: RuntimeAccount | string) {
+    if (typeof account === 'string') {
+      this.account = generateAccount();
+      this.account.name = account;
+      this.address = this.account.addr;
+    } else if (account) {
       // set config if account is passed by user
       this.account = account;
       this.address = account.addr;
```

Comment:
Could you please add `this.account.name = account.name` in `else if` as well? User can pass whole account including name as well, as we're using type `RuntimeAccount`

Reply:
that's not required because this.account's type is `RuntimeAcccount`, so if account has name in it, it will be stored